{"name": "test_greater", "status": "failed", "statusDetails": {"message": "AssertionError: assert 100 > 100", "trace": "@pytest.mark.great\n    def test_greater():\n        num = 100\n>       assert num > 100\nE       assert 100 > 100\n\ntest_compare.py:7: AssertionError"}, "start": 1602843563745, "stop": 1602843563745, "uuid": "bdbdafb3-53b3-4983-b12f-5b01e6361314", "historyId": "ae2daacf9362564e93898689c64e3912", "testCaseId": "9f2619453c551a5ac409fc05ab4afa43", "fullName": "test_compare#test_greater", "labels": [{"name": "tag", "value": "great"}, {"name": "suite", "value": "test_compare"}, {"name": "host", "value": "DESKTOP-PCEHKV8"}, {"name": "thread", "value": "14752-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_compare"}]}